mealie/routes/users/crud.py

```diff
@@ -70,7 +70,12 @@ def update_password(self, password_change: ChangePassword):
             raise HTTPException(
                 status.HTTP_400_BAD_REQUEST, ErrorResponse.respond(self.t("user.ldap-update-password-unavailable"))
             )
-        if not verify_password(password_change.current_password, self.user.password):
+        if (not password_change.current_password) and self.user.is_default_user:
+            # this might be from the setup wizard, which doesn't require the user's current password
+            password_change.current_password = self.settings._DEFAULT_PASSWORD
```

Comment:
Couldn't anyone always change the default users password in this case? Because we're only checking that it's the default user and no current_password was provided. 

Could we just provide the default password as in the request body and leave this code unchanged? 

Maybe I'm missing something

Reply:
> Couldn't anyone always change the default users password in this case?

Yes, but we show the default password at login anyway so we can assume all visitors know the default username and password (until it's changed, because once the email changes it's no longer the default user)

> Could we just provide the default password as in the request body and leave this code unchanged?

...that's a much better solution, lol. I think I had a reason for not doing that but I can't remember what it is, so maybe I'm crazy